common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -38,7 +40,7 @@ public class StatusEffectsOverlay extends Overlay {
     public final Config<Float> fontScale = new Config<>(1.0f);
 
     @Persisted
-    public final Config<Boolean> stackEffects = new Config<>(true);
+    public final Config<StackingBehaviour> effectsStackBehaviour = new Config<>(StackingBehaviour.GROUP);
```

Comment:
Since this is in the StatusEffect overlay, I think it is better to just name this config `stackingBehavior`.

Reply:
Also, you might need a upfixer for this. I'll let @kristofbolyai decide if he thinks we can skip it in this case (since the original config were recently added).

Reply:
We can skip this. I don't think many people disabled it.